internal/ui/ui_nintendosdk.go

```diff
@@ -207,3 +226,7 @@ func (u *userInterfaceImpl) updateIconIfNeeded() {
 func IsScreenTransparentAvailable() bool {
 	return false
 }
+
+func InitialWindowPosition(mw, mh, ww, wh int) (x, y int) {
+	return (mw - ww) / 2, (mh - wh) / 3
```

Comment:
Please avoid copying and pasting the same code...

Reply:
Wanted them to be functionally equivalent, even though they aren't used. I have adjusted to just return 0, 0.

Reply:
I think just moving this function to `ui.go` would sove the issue

Reply:
Hm, that is smarter. Adjusted.